server/server.go

```diff
@@ -311,7 +315,9 @@ func (s *Server) unlimitedDo(pkt *protocol.Packet, connName string) response {
 		return makeRespondResponse(pkt, res)
 
 	case protocol.OpEd25519Sign:
+		loadStart := time.Now()
 		key, err := s.keys.Get(ctx, &pkt.Operation)
+		logKeyLoadDuration(loadStart)
```

Comment:
I think this should be logged after the error check. Logging it here can make latencies appear better or worse than they actually are depending on types of errors, for instance timeout errors could skew the duration much higher, and it would be hard to tell.

Reply:
I don't think we should throw out errors. it should be the time for the log key load function to run. If we're getting a bunch of errors other metrics will go high like the error count. We don't have a great way to granularize metrics because Prometheus metrics are expensive.